semantic_conventions/lib/opentelemetry/semantic_candidates/attributes/aws.rb

```diff
@@ -0,0 +1,293 @@
+    
+      # The JSON-serialized value of each item of the `GlobalSecondaryIndexes` request field
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      GLOBAL_SECONDARY_INDEXES = 'aws.dynamodb.global_secondary_indexes'
+    
+      # The value of the `IndexName` request parameter
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      INDEX_NAME = 'aws.dynamodb.index_name'
+    
+      # The JSON-serialized value of the `ItemCollectionMetrics` response field
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      ITEM_COLLECTION_METRICS = 'aws.dynamodb.item_collection_metrics'
+    
+      # The value of the `Limit` request parameter
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      LIMIT = 'aws.dynamodb.limit'
+    
+      # The JSON-serialized value of each item of the `LocalSecondaryIndexes` request field
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      LOCAL_SECONDARY_INDEXES = 'aws.dynamodb.local_secondary_indexes'
+    
+      # The value of the `ProjectionExpression` request parameter
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      PROJECTION = 'aws.dynamodb.projection'
+    
+      # The value of the `ProvisionedThroughput.ReadCapacityUnits` request parameter
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      PROVISIONED_READ_CAPACITY = 'aws.dynamodb.provisioned_read_capacity'
+    
+      # The value of the `ProvisionedThroughput.WriteCapacityUnits` request parameter
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      PROVISIONED_WRITE_CAPACITY = 'aws.dynamodb.provisioned_write_capacity'
+    
+      # The value of the `ScanIndexForward` request parameter
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      SCAN_FORWARD = 'aws.dynamodb.scan_forward'
+    
+      # The value of the `ScannedCount` response parameter
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      SCANNED_COUNT = 'aws.dynamodb.scanned_count'
+    
+      # The value of the `Segment` request parameter
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      SEGMENT = 'aws.dynamodb.segment'
+    
+      # The value of the `Select` request parameter
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      SELECT = 'aws.dynamodb.select'
+    
+      # The number of items in the `TableNames` response parameter
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      TABLE_COUNT = 'aws.dynamodb.table_count'
+    
+      # The keys in the `RequestItems` object field
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      TABLE_NAMES = 'aws.dynamodb.table_names'
+    
+      # The value of the `TotalSegments` request parameter
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      TOTAL_SEGMENTS = 'aws.dynamodb.total_segments'
+    
+      # The ARN of an [ECS cluster](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/clusters.html)
+      #
+      # @note StabilityLevel.EXPERIMENTAL
+      CLUSTER_ARN = 'aws.ecs.cluster.arn'
```

Comment:
Found one flaw (repeated in several places): using attribute.attr_id in the j2 template is not enough. "ecs" is omitted from the `attr_id`, it's [present in the prefix for the attribute group](https://github.com/open-telemetry/semantic-conventions/blob/v1.26.0/model/registry/aws.yaml#L257-L268). 😭